src/main/xlat.c

```diff
@@ -2216,6 +2219,11 @@ static char *xlat_aprint(TALLOC_CTX *ctx, REQUEST *request, xlat_exp_t const * c
 			snprintf(str, freespace, "%" PRIu64 , request->seq_start);
 			break;
 
+		case 'e': /* Request second */
```

Comment:
%e would be for the actual seconds part of the event timestamp (just like we have %H and %G for the hours and minutes), not the unix timestamp.

> On 25 Apr 2016, at 21:56, Arran Cudbard-Bell <notifications@github.com> wrote:
> 
> In src/main/xlat.c <https://github.com/FreeRADIUS/freeradius-server/pull/1593#discussion_r60977451>:
> 
> > @@ -2216,6 +2219,11 @@ static char *xlat_aprint(TALLOC_CTX *ctx, REQUEST *request, xlat_exp_t const * c
> >  			snprintf(str, freespace, "%" PRIu64 , request->seq_start);
> >  			break;
> >
> > +		case 'e': /* Request second */
> If it really is UTC request second, we already have '%l'
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/FreeRADIUS/freeradius-server/pull/1593/files/4944d0b4aeba693248573b4251c5fa629a897adb#r60977451>